ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -98,21 +98,29 @@ const toggleSavePending = function () {
 };
 on('clicked:saveroll', toggleSavePending);
 
-const numericRollKeys = [
+/**
+ * Ordered list of roll keys (Do not reorganize)
+ */
+const roll_keys = [
```

Comment:
I realized these need to stay the order they were, since that determines the saved state (JSON ordered keys)
I restored these.